```
##[error]Found the following errors:
    Error: While reading versions for port etcd-cpp-apiv3 from file: C:\a\1\s\versions\e-\etcd-cpp-apiv3.json
       File declares version `0.2.12` with SHA: 8ac6dd66dbc8eb4349b1df236e96898e7de654d7
       But local port with the same version has a different SHA: 4e1d07e1e6925a03ada21862eb2483fcac5b91b6
       Please update the port's version fields and then run:

           vcpkg x-add-version etcd-cpp-apiv3
           git add versions
           git commit -m "Update version database"

       to add a new version.
To attempt to resolve all errors at once, run:
vcpkg x-add-version --all
##[error]PowerShell exited with code '1'.
```

The error log is as above, please run the following command and push.

```
./vcpkg format-manifest ports/etcd-cpp-apiv3/vcpkg.json
./vcpkg x-add-version etcd-cpp-apiv3 --overwrite-version
commit and push
```

Explanation of `--overwrite-version`:

After the first modification, please run `./vcpkg x-add-version portName` to generate the corresponding json; for subsequent modifications of the same version, please use `./vcpkg x-add-version portName --overwrite-version` to update git- tree.

--- a/ports/etcd-cpp-apiv3/portfile.cmake
+++ b/ports/etcd-cpp-apiv3/portfile.cmake
@@ -25,3 +25,7 @@ vcpkg_copy_pdbs()
 
 # Handle copyright
 file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/etcd-cpp-apiv3 RENAME copyright)
+
+# Adding usage text
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+

--- a/ports/etcd-cpp-apiv3/usage
+++ b/ports/etcd-cpp-apiv3/usage
@@ -1,0 +1,11 @@
+The package etcd-cpp-apiv3 is compatible with built-in CMake targets:
+
+    find_package(etcd-cpp-api CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE etcd-cpp-api)
+
+In cases where you only need the synchronous runtime and want to avoid the initialization
+of builtin thread pool, use:
+
+    find_package(etcd-cpp-api CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE etcd-cpp-api-core)
+

--- a/versions/e-/etcd-cpp-apiv3.json
+++ b/versions/e-/etcd-cpp-apiv3.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "8ac6dd66dbc8eb4349b1df236e96898e7de654d7",
+      "git-tree": "d11dfdbdc719392013a05c613a68271b167b7ec2",
       "version": "0.2.12",
       "port-version": 0
     }

--- a/versions/e-/etcd-cpp-apiv3.json
+++ b/versions/e-/etcd-cpp-apiv3.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d11dfdbdc719392013a05c613a68271b167b7ec2",
+      "git-tree": "25e86e3e1fbb2d12eb7d32d25f758a6485164834",
       "version": "0.2.12",
       "port-version": 0
     }

